Patch taken from http://sourceforge.net/tracker/?func=detail&aid=2815036&group_id=53614&atid=470969

This is already fixed upstream as well.

--- ruby-gnome2-all-0.19.0/gtkmozembed/extconf.rb.debug	2009-05-18 21:47:19.000000000 +0900
+++ ruby-gnome2-all-0.19.0/gtkmozembed/extconf.rb	2009-06-18 02:22:37.000000000 +0900
@@ -77,7 +77,7 @@
   end
   headers = include_paths.inject([]) do |result, path|
     gtkmozembed_h = File.join(path, "gtkmozembed.h")
-    result += [gtkmozembed_h] if gtkmozembed_h
+    result += [gtkmozembed_h] if FileTest.readable?(gtkmozembed_h)
     result
   end
   glib_mkenums(enum_type_prefix, headers, "GTK_TYPE_", ["gtkmozembed.h"])
